src/containers/Tenant/Schema/SchemaInfoViewer/SchemaInfoViewer.js

```diff
@@ -151,36 +157,43 @@ class SchemaInfoViewer extends React.Component {
             );
         }
 
-        if ([tabletMetricsInfo, partitionConfigInfo, tableStatsInfo.flat()].flat().length === 0) {
-            return <div className={b('item')}>Empty</div>;
+        if (
+            [generalTableInfo, tabletMetricsInfo, partitionConfigInfo, tableStatsInfo.flat()].flat()
+                .length === 0
+        ) {
```

Comment:
I don't think this looks better than before. I understand this is what prettier does, but again, seems like we need to tune it. RN a solution might be to make it a variable